### PR TITLE
Execute queries sequentially

### PR DESCRIPTION
**Pull request by alexdametto**
## Summary of changes
Split queries using `cratedb-sqlparse` and execute queries sequentially.

## Checklist

- [x] Link to issue this PR refers to: https://github.com/crate/cloud/issues/1888
- [x] Relevant changes are reflected in `CHANGES.md`.
- [x] Added or changed code is covered by tests.
- [x] Required Grand Central APIs are already merged.

### CODE REVIEW

File: src/hooks/useExecuteMultiSql.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  // - if a query goes in ERROR, all the following ones will have NOT_EXECUTED status
+  const [queryResults, setQueryResults] = useState<QueryStatus[] | undefined>([]);
+  // The executionId is used to handle the fact that a user can execute a long query
+  // and then execute a new query before the old query finishes;
+  // this is used to avoid UI updating for "old" queries.
+  const executionId = useRef(0);
+
+  const executeSql = useExecuteSql();
+
+  // This is used to update a query status, only if this the latest query executed by the user.
```
Comment (plaharanne) [inline src/hooks/useExecuteMultiSql.ts:29]:
```suggestion
  // This is used to update a query status, only if this is the latest query executed by the user.
```

Comment (alexdametto) [reply src/hooks/useExecuteMultiSql.ts]: Thank you 🙏🏼

File: src/routes/Users/UserInfo.tsx [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
-      if (res.data) {
-        setResult(res.data);
-      }
-    });
-  }, [user]);
+  if (loading || !result) {
+    return <Loader />;
+  }
+
+  console.log(result);
```
Comment (plaharanne) [inline src/routes/Users/UserInfo.tsx:18]: did you forget it?

**Review by plaharanne — APPROVED** [review]
:+1:

File: src/routes/Users/UserInfo.tsx (hunk 2 above)
Comment (alexdametto) [reply src/routes/Users/UserInfo.tsx:18]: Yes 🤦🏼‍♂️

**Pull request merged**